.github/workflows/build-test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - run: yarn install --immutable
       - run: yarn build
       - run: yarn lint
-      - run: yarn test
+      - run: xvfb-run --auto-servernum  yarn test
```

Comment:
@bowensanders Is this how you're running other tests?

Reply:
In metamask-extension, tests are run using `/test/e2e/run-all.js` - which calls `run-e2e-test.js`
that runs `mocha --no-config --timeout 60000 xxx` where xxx is a test in the test/e2e/snaps folder.
I guess this means that we use mocha to run selenium to run individual tests, and in this case they run (locally) in series, not parallel.